File: app/src/main/java/com/cv/anandmjoseph/anandcv/ui/AboutmeOne.java
package com.cv.anandmjoseph.anandcv.ui;


import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.util.DisplayMetrics;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.cv.anandmjoseph.anandcv.R;

/**
 * Created by Anand M Joseph on 12/29/2017.
 */

public class AboutmeOne extends Fragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.about_me_one, container, false);
        return view;
    }
}

File: app/src/main/java/com/cv/anandmjoseph/anandcv/ui/AboutmeOne.kt
package com.cv.anandmjoseph.anandcv.ui


import android.os.Bundle
import android.support.v4.app.Fragment
import android.util.DisplayMetrics
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.cv.anandmjoseph.anandcv.R

/**
 * Created by Anand M Joseph on 12/29/2017.
 */

class AboutmeOne : Fragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.about_me_one, container, false)
    }
}
